### ISSUE TITLE
Cannot find module 'sinon'

### ISSUE THREAD

**Issue by nasht00** [prose/prose#901]
I am trying to install prose locally:

`npm install && mkdir -p dist && gulp`

Results in:

```
localhost:prose nathanh$ npm install && mkdir -p dist && gulp
npm WARN EPEERINVALID sinon-chai@2.8.0 requires a peer of sinon@>=1.4.0 <2 but none was installed.
npm WARN EPACKAGEJSON prose@1.1.16 license should be a valid SPDX license expression
[09:29:15] Using gulpfile ~/Git/tmp/prose/gulpfile.js
[09:29:15] Starting 'templates'...
[09:29:15] Starting 'oauth'...
[09:29:15] Starting 'vendor'...
[09:29:15] Finished 'vendor' after 5.92 ms
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0[09:29:15] Finished 'templates' after 118 ms
100   169  100   169    0     0    192      0 --:--:-- --:--:-- --:--:--   192
[09:29:16] Finished 'oauth' after 924 ms
[09:29:16] Starting 'build-tests'...
[09:29:16] Starting 'build-app'...
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: Cannot find module 'sinon' from '/Users/nathanh/Git/tmp/prose/test'
    at /Users/nathanh/Git/tmp/prose/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/Users/nathanh/Git/tmp/prose/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/Users/nathanh/Git/tmp/prose/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/Users/nathanh/Git/tmp/prose/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/nathanh/Git/tmp/prose/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Users/nathanh/Git/tmp/prose/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:95:15)
```

**Comment by dereklieu** [comment]
@nasht00 I'm having trouble re-producing this. What version of node.js are you running?

**Comment by nasht00** [comment]
```
localhost:~ nathanh$ node --version
v0.12.7
localhost:~ nathanh$ npm --version
3.3.4
```

**Comment by dereklieu** [comment]
Should be fixed by #910. Marking this as closed.
